packages/core/src/components/essentials/tooltip-bar.ts

```diff
@@ -149,6 +152,15 @@ export class TooltipBar extends Tooltip {
 			.node()
 			.getBoundingClientRect();
 		const barPosition = hoveredElement.getBoundingClientRect();
+		const { position: axisPosition } = this.configs;
+		const options = this.model.getOptions();
+		const axisOptions = Tools.getProperty(options, "axes", axisPosition);
+		let truncationThresholdProvided = Tools.getProperty(
```

Comment:
seems like you've set defaults for `threshold` in `Configuration.ts`, so all of these `...Provided` variables need to either be removed or renamed, and any checks for those values should be removed as well since they're always going to exist unless the user sets them to null/undefined.

One check that would be very important is for `truncation.type` so that if it's set to none we do nothing